notebooks/ch-bdqc/delegated-computation.ipynb

```diff
@@ -0,0 +1,472 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# pre-requisites for Blind Delegated quantum computing (BDQC):\n",
+    "\n",
+    "- In the literature, it is common to describe BDQC protocols in the MBQC paradigm, which is why readers need to check the Qiskit textbook for the MBQC tutorial first before proceeding further.\n",
+    "- This tutorial assumes basic knowledge of python, bitwise operations and numpy."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Formal defintion of Blind Delegated quantum computing (BDQC):\n",
+    "\n",
+    "The keyword here is **Blindness**: At the end of the protocol, the server gains no information at all about Alice’s input nor the computation.\n",
+    "\n",
+    "There are various protocols to achieve this, some of them assume Alice can perform quantum measurement at the end but cannot prepare qubits (referred to as receive-and-measure), but we will focus on what is known as prepare-and-send Blind delegated quantum computing.\n",
+    "\n",
+    "## Description Of the protocol\n",
+    "\n",
+    "### Phase 1: preparation phase\n",
+    "\n",
+    "Instead of preparing the resource in the $|+\\rangle^{\\otimes n} $ state for all qubits like we do in MBQC, Alice applies on each qubit $i$ a local random phase shift $\\theta_i$ choosen randomly from $\\{ 0, \\frac{\\pi}{4},\\frac{2\\pi}{4},..., \\frac{7\\pi}{4} \\}$.\n",
```

Comment:
```suggestion
    "Instead of preparing the resource in the $|+\\rangle^{\\otimes n} $ state for all qubits like we do in MBQC, Alice applies on each qubit $i$ a local random phase shift $\\theta_i$ chosen randomly from $\\{ 0, \\frac{\\pi}{4},\\frac{2\\pi}{4},..., \\frac{7\\pi}{4} \\}$.\n",
```